Intersect.Editor/Localization/Strings.cs

```diff
@@ -5011,25 +5015,42 @@ public partial struct ResourceEditor
 
         public static LocalizedString dropdisplay = @"{00} x{01}-{02} | {03}%";
 
-        public static LocalizedString exhaustedgraphic = @"Exhausted Graphic:";
-
         public static LocalizedString folderlabel = @"Folder:";
 
         public static LocalizedString foldertitle = @"Add Folder";
 
         public static LocalizedString folderprompt = @"Enter a name for the folder you'd like to add:";
 
-        public static LocalizedString fromtileset = @"From Tileset";
-
         public static LocalizedString general = @"General";
 
-        public static LocalizedString graphics = @"Graphics";
+        public static LocalizedString Graphics = @"Graphics";
+
+        public static LocalizedString GraphicData = @"Graphic Data";
+
+        public static LocalizedString GraphicFile = @"Graphic File:";
+
+        public static LocalizedString GraphicType = @"Graphic Type:";
+
+        public static Dictionary<ResourceGraphicType, LocalizedString> GraphicTypes = new()
+        {
+            { ResourceGraphicType.Graphic, @"Graphic" },
+            { ResourceGraphicType.Tileset, @"Tileset" },
+            { ResourceGraphicType.Animation, @"Animation" },
+        };
 
         public static LocalizedString harvestevent = @"Event:";
 
-        public static LocalizedString hpregen = @"HP (%);";
+        public static LocalizedString HealthStatesLabel = @"Health States";
+
+        public static LocalizedString HealthStateName = @"Health Graphic State Name:";
+
+        public static LocalizedString HealthStateErrorTitle = @"Invalid Health Graphic State Name";
+
+        public static LocalizedString HealthStateNameError = @"Health Graphic State Name must be unique and greater than 0 characters.";
 
-        public static LocalizedString initialgraphic = @"Initial Graphic:";
+        public static LocalizedString HealthStateNameExists = @"Health Graphic State Name already exists.";
+
+        public static LocalizedString hpregen = @"HP (%);";
```

Comment:
![image](https://github.com/user-attachments/assets/73669b30-f7c8-40e6-a8cb-acffeffb6c64)
This isn't needed, there's a label both above and to the right of the textbox, we only need the one to the right (but you can move it if you want, and we can reword it)